Sources/Workspace/Workspace.swift

```diff
@@ -501,11 +508,11 @@ extension Workspace {
 
         // Abort if we're unable to load the pinsStore or have any diagnostics.
         guard let pinsStore = diagnostics.wrap({ try self.pinsStore.load() }) else {
```

Comment:
This can just be `let pinsStore = try pinsStore.load()`.

Reply:
done